src/main/java/com/bootcamp/demo/models/Booking.java

```diff
@@ -0,0 +1,87 @@
+package com.bootcamp.demo.models;
+import java.util.Date;
+import java.util.Objects;
+
+
+public class Booking {
+    private long id;
+    private long serialNumber;
```

Comment:
serialNumber should be String type

Reply:
serialNumber should be a number because we will map into database

Reply:
But if its number than you have problems if the number is too large and don't fit in your variable, Its more secure to use String bc you don't have this kind of problem. in my pov.